src/NServiceBus.Core/Recoverability/SecondLevelRetries/CustomSecondLevelRetryPolicy.cs

```diff
@@ -14,7 +14,7 @@ public override bool TryGetDelay(IncomingMessage message, Exception ex, int curr
         {
             delay = customRetryPolicy(message);
 
-            return delay != TimeSpan.Zero;
+            return delay != TimeSpan.MinValue;
```

Comment:
@danielmarbach This fixes a bug in v6. v5 and custom policies acceptance tests in v6 use TimeSpan.MinValue. This was discussed with @Particular/nservicebus-maintainers and we decided to piggy back the fix on this PR.